src/components/App/ExpressRecorder.tsx

```diff
@@ -356,21 +367,36 @@ export class ExpressRecorder extends Component<ExpressRecorderProps, State> {
             item.stop();
         });
 
-        this.setState({ doUpload: true });
+        this.setState({
+            doUpload: true
+        });
```

Comment:
```suggestion
        this.setState({
           ...this.state,
           doUpload: true
        });
```
in best practice - if your state is constructed with plenty of keys, update the state using the old one with a spread operator for better state management.